src/yukon/backend/src/devserv/description/example/example_sys_description.json

```diff
@@ -0,0 +1,152 @@
+                                        "value": 4096,
+                                        "default": 4096,
+                                        "min": 4096,
+                                        "max": 4111
+                                }]
+                        },
+                        {
+                                "id": 1,
+                                "name": "esc_front_left",
+                                "health": "OK",
+                                "mode": "OPERATIONAL",
+                                "uptime": 100,
+                                "vendorCode": 62342
+                        },
+                        {
+                                "id": 2,
+                                "name": "esc_front_right",
+                                "health": "CRITICAL",
+                                "mode": "INITIALIZATION",
+                                "uptime": 100,
+                                "vendorCode": 32893
+                        },
+                        {
+                                "id": 3,
+                                "name": "esc_back_left",
+                                "health": "ERROR",
+                                "mode": "MAINTAINANCE",
+                                "uptime": 100,
+                                "vendorCode": 32893
+                        },
+                        {
+                                "id": 4,
+                                "name": "esc_back_right",
+                                "health": "WARNING",
+                                "mode": "SOFTWARE_UPDATE",
+                                "uptime": 100,
+                                "vendorCode": 62342
+                        },
+                        {
+                                "id": 5,
+                                "name": "ms5525_airspeed",
+                                "health": "OK",
+                                "mode": "OFFLINE",
+                                "uptime": 456,
+                                "vendorCode": 8678
+                        },
+                        {
+                                "id": 6,
+                                "name": "servo_right_elevon",
+                                "health": "OK",
+                                "mode": "OPERATIONAL",
+                                "uptime": 45,
+                                "vendorCode": 13434
+                        },
+                        {
+                                "id": 7,
+                                "name": "servo_left_elevon",
+                                "health": "OK",
+                                "mode": "MAINTAINANCE",
+                                "uptime": 456,
+                                "vendorCode": 21431
+                        }
+                ],
+                "grv": [{
+                                "nodeId": 1,
+                                "nodeName": "esc_front_left",
```

Comment:
uavcan.node.GetInfo has a field `name` which becomes ambiguous here. I recommend we name this property "description" or "node description" to avoid the ambiguity. Also, there is no reason to constrain the character set, let's make it "ESC front-left". But, regardless of the prior text, we also need the node name in the GRV.